sway/ipc-json.c

```diff
@@ -384,8 +384,19 @@ static json_object *ipc_json_describe_scratchpad_output(void) {
 
 static void ipc_json_describe_workspace(struct sway_workspace *workspace,
 		json_object *object) {
-	int num = isdigit(workspace->name[0]) ? atoi(workspace->name) : -1;
-
+	int num;
+	if isdigit(workspace->name[0]) {
```

Comment:
There are missing parentheses around the condition. (gcc should not be accepting this...)

Reply:
@RedSoxFan added parentheses. Either way it built and ran so not sure how to catch this next time.

Reply:
Apparently gcc implements isdigit as a macro that starts with a paren, which is why it was accepting it. The FreeBSD CI build (https://builds.sr.ht/~sircmpwn/job/167521), which uses clang, did fail and showed the error. Anyway, thanks for fixing it.